Vagrantfile

```diff
@@ -27,10 +55,11 @@ pushd /arteria/arteria-provisioning/services
 ./install-service $product vagrant vagrant dev $source_path/$product $source_path/arteria /opt/$product 
 popd
 
+sudo /etc/init.d/iptables stop
```

Comment:
Yes. I made all the service on `testarteria1` unreachable. This was the quick-fix solution...